iTerm/README.md

```diff
@@ -1,28 +1,28 @@
->**Note**: Instead of downloading iTerm from the website you can use [Homebrew](http://sourabhbajaj.com/mac-setup/Homebrew/) `brew cask install iterm2`
+You can get the app from [iTerm2 downloads page](http://www.iterm2.com/downloads.html). Once downloaded, drag and drop the **iTerm** application file into your **Applications** folder.
 
-You can download the app from [here](http://www.iterm2.com/). Once downloaded, drag and drop the **iTerm** application file into your **Applications** folder.
+>**Note**: Instead of downloading and installing iTerm2 manually, you can use [Homebrew](http://sourabhbajaj.com/mac-setup/Homebrew/) `brew cask install iterm2`
 
 ## Customization
 ### Colors and Font Settings
 Here are some suggested settings you can change or set, **they are all optional**.
 
-- Set hotkey to open and close the terminal to `command + option + i`
-- Go to profiles -> Default -> Terminal -> Check silence bell to disable the terminal session from making any sound
-- Download [one of iTerm2 color schemes](https://github.com/mbadolato/iTerm2-Color-Schemes/tree/master/schemes) from here. And then set these to your default profile colors.
-- Change the cursor text and cursor color to yellow make it more visible
-- Change the font to 14pt Source Code Pro Lite. Source Code Pro can be downloaded from [here](https://github.com/adobe-fonts/source-code-pro/releases/latest).
-- If you're using BASH instead of ZSH you could add `export CLICOLOR=1` line to your `~/.bash_profile` file for nice coloring of listings
+- set hot-key to open and close the terminal to `command + option + i`
+- go to profiles -> Default -> Terminal -> Check silence bell to disable the terminal session from making any sound
+- download [one of iTerm2 color schemes](https://github.com/mbadolato/iTerm2-Color-Schemes/tree/master/schemes) and then set these to your default profile colors
+- change the cursor text and cursor color to yellow make it more visible
+- change the font to 14pt Source Code Pro Lite. Source Code Pro can be downloaded from [project's github repository](https://github.com/adobe-fonts/source-code-pro/releases/latest).
+- if you're using BASH instead of ZSH you can add `export CLICOLOR=1` line to your `~/.bash_profile` file for nice coloring of listings
 
 [![Screen](https://raw.githubusercontent.com/sb2nov/mac-setup/master/assets/Iterm.png)](https://raw.githubusercontent.com/sb2nov/mac-setup/master/assets/Iterm.png)
 
 ### MacOS shortcuts ⌘←, ⌘→ and ⌥←, ⌥→
-You might be familiar with shortcuts to skip a word (⌥) or go to start/end of the line (⌘), iTerm is not set up to work with these shortcuts by default.
+You might be familiar with shortcuts to skip a word (⌥) or go to start/end of the line (⌘). Of course you can configure iTerm2 to support these shortcuts:
 
-Open up iTerm2 preferences -> Profiles -> Keys -> Click on + icon (add new Keyboard shortcut).
+open up iTerm2 preferences (⌘,) -> Profiles -> Keys -> Click on + icon (add new Keyboard shortcut).
```

Comment:
Why `Open` -> `open`? I'm thinking it should be a capital `O` since it's a new sentence.

Reply:
Because previous statement has `colon` in the end:

something like this :)

Reply:
Ok, makes sense :) Another reason for a style guide where we have these things defined!